packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/Array/Examples.tsx

```diff
@@ -205,7 +205,7 @@ export const ArrayFromFormHandler = () => {
                 </Field.Composition>
 
                 <Iterate.Toolbar>
-                  <Iterate.RemoveButton />
+                  <Iterate.RemoveButton confirmRemove />
```

Comment:
```suggestion
                  <Iterate.RemoveButton confirm />
```

Can it just be confirm? As the component is RemoveButton, so we sort of know the confirming should be about removing. Hence the name of the component.

Reply:
Yes 👍

Reply:
But on the other hand like more unique props so it's easier to maintain for everyone. 

Could be showConfirmDialog or confirmBeforeRemove

Any thoughts about such prop names?

Reply:
Sure, perhaps just `showConfirmDialog`, I think I like that the best.